state/converters.go

```diff
@@ -176,25 +176,24 @@ func (s *State) convertToProcessTransactionResponse(responses []*executor.Proces
 				return nil, err
 			}
 		} else {
-			log.Warnf("ProcessTransactionResponse[GetRlpTx]: empty for tx %v", result.TxHash)
+			log.Warnf("Invalid RLP (tx hash may be empty) for tx %v", result.TxHash)
```

Comment:
what about printing the rlp here? Is the result.TxHash needed?? If the full rlp is wrong ithink we won't have the txHash